tomcat/src/main/tomcat-base/conf/catalina.properties

```diff
@@ -157,3 +157,6 @@ session.DatastoreStore.sessionKind=TomcatGCloudSession
 
 # Defines the maximum time (in seconds) a session can be inactive before being deleted by the expiration process.
 session.DatastoreStore.sessionMaxInactiveTime=3600
+
+gcp.stackdriver-trace.scheduledDelay=15
+gcp.project=""
```

Comment:
What is the default behavior if this value is not set? Is the trace module able to use the auth credentials in the Flex environment?